pywinauto/unittests/test_UIAWrapper.py

```diff
@@ -40,10 +40,9 @@ class UIAWrapperTests(unittest.TestCase):
         def setUp(self):
             """Start the application set some data and ensure the application
             is in the state we want it."""
-            backend.activate("uia")
 
             # start the application
-            self.app = Application().Start(os.path.join(wpf_samples_folder, u"WpfApplication1.exe"))
+            self.app = Application(backend = 'uia').Start(os.path.join(wpf_samples_folder, u"WpfApplication1.exe"), timeout = 30)
```

Comment:
I think 79 chars is too old limit because nobody goes to print a code out. Splitting line makes sense to me when it's longer than 100-120 chars. Especially for test code.